adam-core/src/main/scala/org/bdgenomics/adam/rdd/features/FeatureParser.scala

```diff
@@ -196,8 +205,16 @@ class IntervalListParser extends Serializable {
 
 class BEDParser extends FeatureParser {
 
+  def isHeader(line: String): Boolean = {
+    return line.startsWith("#") || line.startsWith("browser") || line.startsWith("track")
+  }
+
   override def parse(line: String): Seq[Feature] = {
 
+    if (isHeader(line)) {
+      return Seq()
```

Comment:
here I find it better to short circuit return than to put the whole method in an if block